MyNUnit/Attributes/After.cs

```diff
@@ -0,0 +1,10 @@
+namespace Attributes;
+
+/// <summary>
+/// Attribute for methods, that are called after the test
```

Comment:
```suggestion
/// Attribute for methods that are called after the test
```
В английском вроде без запятой перед that